TTTAttributedLabel/TTTAttributedLabel.m

```diff
@@ -848,6 +848,7 @@ - (CGRect)boundingRectForCharacterRange:(NSRange)range {
     [textStorage addLayoutManager:layoutManager];
 
     NSTextContainer *textContainer = [[NSTextContainer alloc] initWithSize:self.bounds.size];
+    textContainer.lineFragmentPadding = 0;
```

Comment:
Oh we forked this pod hah! I was just about to say 🤔  :) 

You going to return this fix to it's other real home too? :)

Reply:
> Oh we forked this pod hah! I was just about to say 🤔 :)

ha i also had no idea we'd forked it until i noticed that the TTTAttributedLabel i was pulling into StarterKit to help debug was different 😬 

> You going to return this fix to it's other (real) home too? :)

i would totally consider this normally, but upstream hasn't merged a change to master since January 2017... and there are a bunch of pull requests open with no response, so it's unlikely the change would get in. it might still be worth opening an issue so that it's tracked in case someone else stumbles upon the same issue, but i'm not sure if it's worth going to the trouble to do that given how dead the library seems. but i'll do it if i have some time!